packages/react-static-plugin-css-modules/src/node.api.js

```diff
@@ -0,0 +1,35 @@
+import ExtractCssChunks from 'extract-css-chunks-webpack-plugin'
+
+export default (options = {}) => ({
+  webpack: (config, { stage }) => {
+    let loaders = []
+
+    const styleLoader = { loader: 'style-loader' }
+    const cssLoader = {
+      loader: 'css-loader',
+      options: Object.assign({
```

Comment:
Since we're already using spread in the node `loaders` below, would it make sense to use it here?

Reply:
good point